src/covid_model_seiir_pipeline/forecasting/model/forecast_metrics.py

```diff
@@ -10,11 +10,12 @@ def compute_output_metrics(infection_data: pd.DataFrame,
                            components_past: pd.DataFrame,
                            components_forecast: pd.DataFrame,
                            seir_params: Dict[str, float],
-                           compartment_info: CompartmentInfo) -> Tuple[pd.DataFrame, pd.DataFrame, pd.DataFrame, pd.DataFrame]:
+                           compartment_info: CompartmentInfo) -> Tuple[pd.DataFrame, pd.DataFrame,
+                                                                       pd.DataFrame, pd.DataFrame]:
```

Comment:
I know this really didn't change, but consider defining at least namedtuples for this output in the future.